### DISCUSSION TITLE
Mode会从Rules自动跳到Global

### DISCUSSION THREAD

**Discussion by lesnolie** [kkkgo/PaoPaoGateWay#70] [问和答]
您好，您的产品实在是太好用了
我搜索了所有的discussion，好像没有人提这个bug
我在yaml中填写了rules
所以现在mode这里，我的rules会自动变成global
不知道哪里出问题了？
您能帮忙解答一下吗，谢谢

**Comment by kkkgo** [comment]
开启`fast_node`功能会忽略所有规则。

**Reply by lesnolie** [reply]
原来是这样，还是看规则看漏了，谢谢解答。
